### PR TITLE
Display MOTD in cli

### PR DESCRIPTION
**Pull request by valyo**
Display in cli. 
Works with https://github.com/ScilifelabDataCentre/dds_web/pull/1198

Please include the following in this section

- [ ] Summary of the changes and the related issue
- [ ] Relevant motivation and context
- [ ] Any dependencies that are required for this change

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

Please delete options that are not relevant.

- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Rebase/merge the branch which this PR is made to
- [ ] Product Owner / Scrum Master: This PR is made to the `master` branch and I have updated the [version](../dds_cli/version.py)

## Formatting and documentation

- [ ] I have added a row in the [changelog](../CHANGELOG.md)
- [ ] The code follows the style guidelines of this project: Black / Prettier formatting
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

## Tests

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: dds_cli/__main__.py [hunk 1]
```diff
@@ -91,6 +91,20 @@
...
+    dds_cli.utils.stderr_console.print("[bold]Important information:", motd.get("message"), "\n")
+except (
+    dds_cli.exceptions.AuthenticationError,
+    dds_cli.exceptions.ApiResponseError,
+    dds_cli.exceptions.ApiRequestError,
+    dds_cli.exceptions.DDSCLIException,
+) as err:
+    LOG.error(err)
+    sys.exit(1)
+
```
Comment (inaod568) [inline dds_cli/__main__.py:107]: I think we should only have: 
```
motd = dds_cli.motd_manager.MotdManager.get_latest_motd()
```
And then you can structure that function as the `get_user_name_if_logged_in` - just use `try except` without any specific exceptions. 
Then do
```
if motd:
    dds_cli.utils.stderr_console.print("[bold]Important information:", motd.get("message"), "\n")
```

So: 
```suggestion
motd = dds_cli.motd_manager.MotdManager.get_latest_motd()
if motd:
    dds_cli.utils.stderr_console.print(f"[bold]Important information: {motd} \n")
```

File: dds_cli/motd_manager.py [hunk 2]
```diff
@@ -63,11 +63,22 @@ def __init__(
...
+    def get_latest_motd(self):
+        """Get the latest MOTD from dabase"""
+        response_json, _ = dds_cli.utils.perform_request(
+            endpoint=DDSEndpoint.MOTD,
+            headers=self.token,
+            method="get",
+            error_message="Failed getting MOTD",
+        )
+
+        return response_json
```
Comment (inaod568) [inline dds_cli/motd_manager.py:84]: This is a continuation of the previous comment
```suggestion
        try:
            response_json, _ = dds_cli.utils.perform_request(
                endpoint=DDSEndpoint.MOTD,
                headers=self.token,
                method="get",
                error_message="Failed getting MOTD",
            )
        except: 
            pass
        return response_json.get("message")
```

Comment (inaod568) [reply dds_cli/motd_manager.py:84]: Reason for this btw: If something happens with specifically MOTD Endpoint it should still be ppossible to continue the delivery

Comment (valyo) [reply dds_cli/motd_manager.py:84]: aha, you mean in case the command is for data upload

File: tests/test_utils.py [hunk 3]
```diff
@@ -424,10 +424,10 @@ def test_perform_request_add_motd_error_insufficient_credentials() -> None:
         "email": "",
     }
     with Mocker() as mock:
-        mock.post(DDSEndpoint.ADD_NEW_MOTD, status_code=403, json=response_json)
+        mock.post(DDSEndpoint.MOTD, status_code=403, json=response_json)
```
Comment (inaod568) [inline tests/test_utils.py:427]: Not related to this, but instead line 335 and 336 I believe (the test that fails)
Change the two asserts to only:
```
assert str(exc_info.value) == "[Errno Expecting value] str: 0"
```

**Review by inaod568 — CHANGES_REQUESTED** [review]

**Comment by valyo** [comment]
finally, and for the record - I didn't understand what it is for either :)

**Comment by inaod568** [comment]
Merged this now so that we're sure we have it. Forgot to point out the changelog, will add directly to dev. Also, tests for this can be added in https://github.com/ScilifelabDataCentre/dds_cli/pull/499

**Pull request merged**